ci/.gitlab-ci.yml

```
@@ -19,7 +19,8 @@ stages:
     - trying
   variables:
     GIT_SUBMODULE_STRATEGY: recursive
-    SPACK_SHA: 08efe13efb627793fbe048542890ee85668c8100
+    # Custom spack sha used to make a umpire fix + pika 0.9.0 available
```

pika 0.9.0 is already available on develop without the extra patches. Maybe like this:
```suggestion
    # Custom spack sha used to make an umpire fix available
    # (https://github.com/aurianer/spack/commit/e4f4b70ed153167c06aa63eefa41d975394af066)
```